staging/src/k8s.io/cloud-provider/options/options_test.go

```diff
@@ -84,6 +85,11 @@ func TestDefaultFlags(t *testing.T) {
 				},
 			},
 		},
+		NodeController: &NodeControllerOptions{
```

Comment:
Do we have a test for the backward compatible case, where no node config is provided and we ensure workerCount is set to 1?

Reply:
Isn't it tested right here? If I understand correctly this test checks config if no flags are added so this would be backward compatible case.